CHANGELOG.md

```diff
@@ -1,3 +1,131 @@
+# 0.26.0 (2018-08-??)
+
+## Language changes
+
+- **(breaking-change)** Revert do not collapse unions for sibling types. ([#6351], thanks @asterite)
+- **(breaking-change)** Restrict `exp.@var` to types in the same hierarchy. ([#6358], thanks @asterite)
+- **(breaking-change)** Constant lookup context in macro is now lexical. ([#5354], thanks @MakeNowJust)
+- **(breaking-change)** Evaluate instance var initializers at the metaclass level (ie: disallow using `self`). ([#6414], thanks @asterite)
+- Add `&+` `&-` `&*` `&**` operators parsing. NB: No behaviour is assigned to these operators yet. ([#6329], thanks @bcardiff)
+- Add support for empty `case` without `when`. ([#6367], thanks @straight-shoota)
+
+## Standard library
+
+### Macros
+
+- Add `pp!` and `p!` macro methods. ([#6374], thanks @straight-shoota)
```

Comment:
I don't think this change lies in the standard library section, as you can't add or change the available macro methods

Reply:
True. But macros are also documented in stdlib API. I'm not sure about this.

But it reminds me that I actually forgot adding docs for `p! and `pp!` in that PR :o

Reply:
Yeah, I'll move it to language changes. Macro methods are defined in the language itself at the end of the day. But I will keep a Macros subsection for that.